docs/examples-setup.md

```diff
@@ -1,21 +1,24 @@
 # Examples Setup
+Install dependencies for iOS and Android
 
-## iOS
+    cd example
+    npm install
 
+## iOS
 1. Install dependencies and open the workspace:
 
-    cd example
-    npm install
+    ```
     cd ios
     pod install
     open AirMapsExplorer.xcworkspace
+    ```
+1. Make sure the `AirMapsExplorer` target is selected and click `Run`
 
-2. Make sure the `AirMapsExplorer` target is selected and click `Run`
-
-## android
-
+## Android
 1. Start your emulator
+1. Install via Gradle:
 
-2. Install via gradle:
-
-     ./example/android/gradlew installDebug
+    ```
+    react-native run-android
+    ./example/android/gradlew installDebug
```

Comment:
instead of these two commands, i think it is more correct to `cd` into `example/` and then run `react-native run-android`